### PR TITLE
PERF-3788 Use multi-threading to speed up Loader in ExpressiveQueries workload

### PR DESCRIPTION
**Pull request by Fefer-Ivan**
There is a TODO to use multithreading to load documents faster in ExpressiveQueries workload. This now can be fixed using MultipleThreadsPerCollection option.

### CODE REVIEW

File: src/workloads/query/ExpressiveQueries.yml [hunk 1]
```diff
@@ -19,15 +19,13 @@ Actors:
...
-  # TODO: TIG-2938 Use multiple threads to speedup insertion.
-  Threads: 1
+  Threads: 4
   Phases:
   - Repeat: 1
     Database: &DB test
-    Threads: 1
+    MultipleThreadsPerCollection: true
     CollectionCount: 1  # Collection name will be Collection0, this is not configurable.
-    DocumentCount: 1e6
```
Comment (mtandrei) [inline src/workloads/query/ExpressiveQueries.yml:30]: [optional nit] I would revert this change as it's easier to read '6' than to count six zeros.

**Review by mtandrei — DISMISSED** [review]
LGTM; thanks Ivan!

File: src/workloads/query/ExpressiveQueries.yml (hunk 1 above)
Comment (Fefer-Ivan) [reply src/workloads/query/ExpressiveQueries.yml:30]: I agree, but for when MultipleThreadsPerCollection is true, yaml parsing is forcing this number to be an integer because it is doing integer division of DocumentCount / ThreadCount. 

We can address it in C++ code, but to keep the PR smaller, I decided to just change it to integer.

Comment (Fefer-Ivan) [reply src/workloads/query/ExpressiveQueries.yml:30]: The fix for it is actually pretty easy, but I would like to do it in a separate PR

**Comment by Fefer-Ivan** [comment]
https://spruce.mongodb.com/version/63d948bd3e8e86193aeae1fa/tasks

File: src/workloads/query/ExpressiveQueries.yml (hunk 1 above)
Comment (mtandrei) [reply src/workloads/query/ExpressiveQueries.yml:30]: Sounds good; thanks for addressing this!

**Review by mtandrei — APPROVED** [review]
Still LGTM; thanks Ivan!
